lib/src/test/kotlin/main/StackTests.kt

```diff
@@ -0,0 +1,25 @@
+/*
+ * This Kotlin source file was generated by the Gradle 'init' task.
+ */
+package main
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+
+class StackTests {
+    @Test
+    fun checkPushAddsElements() {
+        val stack = Stack<Int>()
+        assertThat(stack.size()).isEqualTo(0)
+        assertThat(stack.isEmpty()).isTrue
+
+        for (i in 1..10) {
```

Comment:
@thebongy Could you write another test with another data type? Preferably, a custom data type?

Reply:
On it @ashikka

Reply:
Morever, @thebongy please do not PR to the `main` branch. Kindly PR to the respective data structure branches.

Reply:
My bad, I have switched to the stack branch

Reply:
@ashikka I have added a test with a custom data type, can you please review it?

Reply:
Great job! @thebongy :sparkling_heart: